.circleci/config.yml

```diff
@@ -101,12 +101,15 @@ executors:
   puppeteer-executor:
     docker:
       - image: << pipeline.parameters.workbench-image >>
+        environment:
+          TZ: "America/New_York"
```

Comment:
Set timezone in CircleCi VM docker images. Wrong time displayed for the wokspace last changed time.

When test ran around `[[6/11/2021, 12:03:41] `, displayed workspaces' last changed time were future time.
<img width="445" alt="Screen Shot 2021-06-11 at 3 45 47 PM" src="https://user-images.githubusercontent.com/35533885/121741232-3ec26b00-cacc-11eb-8fdc-b289b008bbfb.png">